src/datasource.ts

```diff
@@ -348,7 +346,7 @@ export function filterEmptyQueryTargets(targets: QueryTarget[]): QueryTarget[] {
   return targets.filter(target => {
     if (target && !target.hide) {
       if (target.tab === Timeseries || target.tab === undefined) {
-        return target.target && target.target !== 'Start typing tag id here';
```

Comment:
yes, cause now it will equal to the number (`id` actually) or undefined

Reply:
rn now I see places like `"target": "Start typing tag id here",` in JSON
can we then filter it out somewhere? or we already doing it?

Reply:
Yes, we can have it in JSON. But this field means that nothing is selected for Timeseries tab. Whit code changes in PR we will send wrong formatted list request, which returns 400. But after user chose some option from a list he will never be able to write "Start typing tag id here" to JSON. We can remove all targets that include this string in JSON. But I would like to avoid this string mentioning inside code.

Reply:
but it'll return 400, meaning that redish alert will pop, that's not really nice. considering that it's a default value for all old panels..

Reply:
alert will not pop up, I've checked. But I agree that having failed network request is not good and could be annoying in the console. We can get rid of it by migration script, just removing target with such value

Reply:
yeah, that we can do

Reply:
should we remove
```
  defaults = {
    target: 'Start typing tag id here',
```
then? make a placeholder instead?

Reply:
oh, yes, that's right. Didn't think about it, doing